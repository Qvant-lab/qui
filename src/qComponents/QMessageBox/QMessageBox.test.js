import Component from './src/QMessageBox.vue';

describe('QMessageBox', () => {
  it('should match snapshot', () => {
    const { element } = shallowMount(Component);
    expect(element).toMatchSnapshot();
  });

  it('data should match snapshot', () => {
    expect(Component.data()).toMatchSnapshot();
  });
});
