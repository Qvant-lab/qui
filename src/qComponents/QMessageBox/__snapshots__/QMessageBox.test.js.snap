// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QMessageBox data should match snapshot 1`] = `
Object {
  "callback": null,
  "focusAfterClosed": null,
  "isCancelBtnLoading": false,
  "isComponentUsed": false,
  "isConfirmBtnLoading": false,
  "isShown": false,
}
`;

exports[`QMessageBox should match snapshot 1`] = `
<transition-stub
  name="q-msgbox-fade"
>
  <!---->
</transition-stub>
`;
